src/envoy/http/service_control/handler_impl.cc

```diff
@@ -54,6 +56,32 @@ const Envoy::Http::LowerCaseString kAndroidCertHeader{"x-android-cert"};
 constexpr char JwtPayloadIssuerPath[] = "iss";
 constexpr char JwtPayloadAudiencePath[] = "aud";
 
+std::string CheckErrorToRcDetail(const ScResponseError& error) {
+  return error.is_network_error
+             ? absl::StrCat("service_control_check_call_failure{", error.name,
```

Comment:
I like this logic, just a small NIT on naming. Maybe `service_control_check_network_failure`?